packages/utils/telemetry-utils/src/events.ts

```diff
@@ -3,6 +3,8 @@
  * Licensed under the MIT License.
  */
 
+// False positive: this is an import from the `events` package, not from Node.
+// eslint-disable-next-line unicorn/prefer-node-protocol
 import { EventEmitter } from "events";
```

Comment:
How sure are you about this?
The emitted .d.ts code seems to say:
```typescript
/// <reference types="node" resolution-mode="require"/>
import { EventEmitter } from "events";
```

Reply:
According to the `events`' docs, it appears to be basically an browser-compatible adapter for Node's events library. So it wouldn't surprise me if the corresponding types library basically re-exported Node's types. But I don't know for sure.
https://www.npmjs.com/package/events

Reply:
Got it. We don't appear to need this lint disable in most other places that import the events package. I see three lint disables. Repo is certainly not uniform so perhaps other places are missing the rule.
I looked at container-loader audience.ts and the emitted .d.ts does also say `/// <reference types="node"`.
Next, I'll try to debug under node and under browser to see what is loaded.
This is coming up because `/// <reference types="node"` is not supported with tsc moduleResolution Bundler, that we expect lots of customers to be working with.
cc @DLehenbauer